src/api/public/apidocs-new/OBS-v2.10.50.yaml

```
@@ -19,6 +19,7 @@ tags:
   - name: Request
   - name: Trigger
   - name: Workers
+  - name: Sources - Projects
```

```suggestion
  - name: Request
  - name: Sources - Projects
  - name: Trigger
  - name: Workers
```
Just to maintain the list alphabetically sorted.